### PR TITLE
Treewide: remove inactive maintainer viric

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Description of changes

https://github.com/NixOS/nixpkgs/issues/290642

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AndersonTorres** [comment]
For the future merger:
Can we relax the restriction here and merge it ignoring the formatter?

**Review by Mic92 — APPROVED** [review]
Fine with me let's wait 1-2 days for a response.

**Review by viric — APPROVED** [review]
I confirm I'm an inactive maintainer.

**Comment by Mic92** [comment]
Okay. Than we can merge it right away.

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by tobim** [comment]
I want to add myself for `perf`, `ghostscript`, and `logrotate`. Can you make that change or should I open a follow up PR?

**Comment by AndersonTorres** [comment]
> I want to add myself for `perf`, `ghostscript`, and `logrotate`. Can you make that change or should I open a follow up PR?

Better:
I will remove them from this PR. This way you can adopt them right now.

**Comment by Aleksanaa** [comment]
As the first person to encounter the unused with error due to this, can you share your thoughts on this? 😄🎙️

File: pkgs/applications/audio/lingot/default.nix [hunk 1]
```diff
@@ -46,6 +46,6 @@ stdenv.mkDerivation rec {
     homepage = "https://www.nongnu.org/lingot/";
     license = lib.licenses.gpl2Plus;
     platforms = with lib.platforms; linux;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/audio/lingot/default.nix:49]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/graphics/meshlab/default.nix [hunk 2]
```diff
@@ -95,7 +95,7 @@ mkDerivation rec {
     mainProgram = "meshlab";
     homepage = "https://www.meshlab.net/";
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/graphics/meshlab/default.nix:98]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/graphics/wings/default.nix [hunk 3]
```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
     homepage = "http://www.wings3d.com/";
     description = "Subdivision modeler inspired by Nendo and Mirai from Izware";
     license = lib.licenses.tcltk;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/graphics/wings/default.nix:51]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/misc/librecad/default.nix [hunk 4]
```diff
@@ -72,7 +72,7 @@ mkDerivation rec {
     description = "2D CAD package based on Qt";
     homepage = "https://librecad.org";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/misc/librecad/default.nix:75]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/networking/instant-messengers/silc-client/default.nix [hunk 5]
```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
     description = "Secure Internet Live Conferencing server";
     mainProgram = "silc";
     license = lib.licenses.gpl2;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/networking/instant-messengers/silc-client/default.nix:36]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/networking/irc/irssi/fish/default.nix [hunk 6]
```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     homepage = "https://github.com/falsovsky/FiSH-irssi";
     license = licenses.mit;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/networking/irc/irssi/fish/default.nix:25]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/networking/offrss/default.nix [hunk 7]
```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
     homepage = "http://vicerveza.homeunix.net/~viric/cgi-bin/offrss";
     description = "Offline RSS/Atom reader";
     license = licenses.agpl3Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/networking/offrss/default.nix:39]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/networking/siproxd/default.nix [hunk 8]
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     homepage = "http://siproxd.sourceforge.net/";
     description = "Masquerading SIP Proxy Server";
     mainProgram = "siproxd";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/networking/siproxd/default.nix:20]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/office/mmex/default.nix [hunk 9]
```diff
@@ -71,7 +71,7 @@ stdenv.mkDerivation rec {
     description = "Easy-to-use personal finance software";
     homepage = "https://www.moneymanagerex.org/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/office/mmex/default.nix:74]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/science/electronics/caneda/default.nix [hunk 10]
```diff
@@ -19,7 +19,7 @@ mkDerivation rec {
     mainProgram = "caneda";
     homepage = "http://caneda.org";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/science/electronics/caneda/default.nix:22]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/science/electronics/xoscope/default.nix [hunk 11]
```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
     mainProgram = "xoscope";
     homepage = "https://xoscope.sourceforge.net";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/science/electronics/xoscope/default.nix:30]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/science/math/yacas/default.nix [hunk 12]
```diff
@@ -80,7 +80,7 @@ stdenv.mkDerivation rec {
     description = "Easy to use, general purpose Computer Algebra System${lib.optionalString enableGui ", built with GUI."}";
     homepage = "http://www.yacas.org/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/science/math/yacas/default.nix:83]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/search/doodle/default.nix [hunk 13]
```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
     homepage = "https://grothoff.org/christian/doodle/";
     description = "Tool to quickly index and search documents on a computer";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/applications/search/doodle/default.nix:18]:
```suggestion
    maintainers = [ ];
```

File: pkgs/by-name/zs/zsync/package.nix [hunk 14]
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     homepage = "http://zsync.moria.org.uk/";
     description = "File distribution system using the rsync algorithm";
     license = licenses.artistic2;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/by-name/zs/zsync/package.nix:24]:
```suggestion
    maintainers = [ ];
```

File: pkgs/development/libraries/flann/default.nix [hunk 15]
```diff
@@ -73,7 +73,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/flann-lib/flann";
     license = lib.licenses.bsd3;
     description = "Fast approximate nearest neighbor searches in high dimensional spaces";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/development/libraries/flann/default.nix:76]:
```suggestion
    maintainers = [ ];
```

File: pkgs/development/libraries/vxl/default.nix [hunk 16]
```diff
@@ -44,7 +44,7 @@ stdenv.mkDerivation (finalAttrs: {
     # license appears contradictory; see https://github.com/vxl/vxl/issues/752
     # (and see https://github.com/InsightSoftwareConsortium/ITK/pull/1920/files for potential patch)
     license = [ lib.licenses.unfree ];
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/development/libraries/vxl/default.nix:47]:
```suggestion
    maintainers = [ ];
```

File: pkgs/development/python-modules/aiohttp-apispec/default.nix [hunk 17]
```diff
@@ -45,6 +45,6 @@ buildPythonPackage rec {
     description = "Build and document REST APIs with aiohttp and apispec";
     homepage = "https://github.com/maximdanilchenko/aiohttp-apispec/";
     license = licenses.mit;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/development/python-modules/aiohttp-apispec/default.nix:48]:
```suggestion
    maintainers = [ ];
```

File: pkgs/development/tools/misc/cbrowser/default.nix [hunk 18]
```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
 
     homepage = "https://sourceforge.net/projects/cbrowser/";
 
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/development/tools/misc/cbrowser/default.nix:33]:
```suggestion
    maintainers = [ ];
```

File: pkgs/development/tools/misc/cscope/default.nix [hunk 19]
```diff
@@ -47,7 +47,7 @@ stdenv.mkDerivation rec {
 
     homepage = "https://cscope.sourceforge.net/";
 
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/development/tools/misc/cscope/default.nix:50]:
```suggestion
    maintainers = [ ];
```

File: pkgs/development/tools/profiling/malt/default.nix [hunk 20]
```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
     description = "Memory tool to find where you allocate your memory";
     homepage = "https://github.com/memtt/malt";
     license = licenses.cecill-c;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/development/tools/profiling/malt/default.nix:31]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/bsdgames/default.nix [hunk 21]
```diff
@@ -63,7 +63,7 @@ stdenv.mkDerivation rec {
     homepage = "http://www.t2-project.org/packages/bsd-games.html";
     description = "Ports of all the games from NetBSD-current that are free";
     license = lib.licenses.free;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/bsdgames/default.nix:66]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/gltron/default.nix [hunk 22]
```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
     description = "Game based on the movie Tron";
     mainProgram = "gltron";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/gltron/default.nix:28]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/neverball/default.nix [hunk 23]
```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
     homepage = "https://neverball.org/";
     description = "Tilt the floor to roll a ball";
     license = "GPL";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/neverball/default.nix:48]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/pioneers/default.nix [hunk 24]
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     description = "Addicting game based on The Settlers of Catan";
     homepage = "https://pio.sourceforge.net/";  # https does not work
     license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/pioneers/default.nix:27]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/torcs/default.nix [hunk 25]
```diff
@@ -49,7 +49,7 @@ stdenv.mkDerivation rec {
     description = "Car racing game";
     homepage = "https://torcs.sourceforge.net/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/torcs/default.nix:52]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/trigger/default.nix [hunk 26]
```diff
@@ -65,7 +65,7 @@ stdenv.mkDerivation rec {
     mainProgram = "trigger-rally";
     homepage = "http://trigger-rally.sourceforge.net/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/trigger/default.nix:68]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/ufoai/default.nix [hunk 27]
```diff
@@ -43,7 +43,7 @@ stdenv.mkDerivation rec {
     homepage = "http://ufoai.org";
     description = "Squad-based tactical strategy game in the tradition of X-Com";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/ufoai/default.nix:46]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/ultimatestunts/default.nix [hunk 28]
```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
     homepage = "http://www.ultimatestunts.nl/";
     description = "Remake of the popular racing DOS-game Stunts";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/ultimatestunts/default.nix:23]:
```suggestion
    maintainers = [ ];
```

File: pkgs/games/vdrift/default.nix [hunk 29]
```diff
@@ -52,7 +52,7 @@ let
       mainProgram = "vdrift";
       homepage = "http://vdrift.net/";
       license = lib.licenses.gpl2Plus;
-      maintainers = with lib.maintainers; [ viric ];
+      maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/games/vdrift/default.nix:55]:
```suggestion
      maintainers = [ ];
```

File: pkgs/os-specific/linux/netatop/default.nix [hunk 30]
```diff
@@ -51,6 +51,6 @@ stdenv.mkDerivation {
     homepage = "https://www.atoptool.nl/downloadnetatop.php";
     license = lib.licenses.gpl2Only;
     platforms = lib.platforms.linux;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/os-specific/linux/netatop/default.nix:54]:
```suggestion
    maintainers = [ ];
```

File: pkgs/servers/silc-server/default.nix [hunk 31]
```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
     description = "Secure Internet Live Conferencing server";
     mainProgram = "silcd";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/servers/silc-server/default.nix:17]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/X11/xbindkeys/default.nix [hunk 32]
```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.nongnu.org/xbindkeys/xbindkeys.html";
     description = "Launch shell commands with your keyboard or your mouse under X Window";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/X11/xbindkeys/default.nix:18]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/X11/xdotool/default.nix [hunk 33]
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.semicomplete.com/projects/xdotool/";
     description = "Fake keyboard/mouse input, window management, and more";
     license = lib.licenses.bsd3;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/X11/xdotool/default.nix:27]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/X11/xtrace/default.nix [hunk 34]
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     homepage = "https://salsa.debian.org/debian/xtrace";
     description = "Tool to trace X11 protocol connections";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/X11/xtrace/default.nix:27]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/admin/tigervnc/default.nix [hunk 35]
```diff
@@ -153,7 +153,7 @@ stdenv.mkDerivation rec {
     homepage = "https://tigervnc.org/";
     license = lib.licenses.gpl2Plus;
     description = "Fork of tightVNC, made in cooperation with VirtualGL";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/admin/tigervnc/default.nix:156]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/backup/btar/default.nix [hunk 36]
```diff
@@ -32,6 +32,6 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl3Plus;
     homepage = "https://viric.name/cgi-bin/btar";
     platforms = platforms.all;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/backup/btar/default.nix:35]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/compression/pbzip2/default.nix [hunk 37]
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     homepage = "http://compression.ca/pbzip2/";
     description = "Parallel implementation of bzip2 for multi-core machines";
     license = licenses.bsd2;
-    maintainers = with maintainers; [viric];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/compression/pbzip2/default.nix:27]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/filesystems/mtdutils/default.nix [hunk 38]
```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
     downloadPage = "https://git.infradead.org/mtd-utils.git";
     license = licenses.gpl2Plus;
     homepage = "http://www.linux-mtd.infradead.org/";
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/filesystems/mtdutils/default.nix:42]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/graphics/exiftags/default.nix [hunk 39]
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     homepage = "http://johnst.org/sw/exiftags/";
     description = "Displays EXIF data from JPEG files";
     license = lib.licenses.free;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/graphics/exiftags/default.nix:24]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/misc/pal/default.nix [hunk 40]
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     homepage = "https://palcal.sourceforge.net/";
     description = "Command-line calendar program that can keep track of events";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/misc/pal/default.nix:27]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/misc/xburst-tools/default.nix [hunk 41]
```diff
@@ -43,7 +43,7 @@ stdenv.mkDerivation {
     description = "Qi tools to access the Ben Nanonote USB_BOOT mode";
     license = lib.licenses.gpl3;
     homepage = "http://www.linux-mtd.infradead.org/";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/misc/xburst-tools/default.nix:46]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/networking/filegive/default.nix [hunk 42]
```diff
@@ -18,7 +18,7 @@ buildGoModule rec {
     homepage = "https://viric.name/cgi-bin/filegive";
     description = "Easy p2p file sending program";
     license = licenses.agpl3Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/networking/filegive/default.nix:21]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/networking/mosh/default.nix [hunk 43]
```diff
@@ -59,7 +59,7 @@ stdenv.mkDerivation rec {
       especially over Wi-Fi, cellular, and long-distance links.
     '';
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/networking/mosh/default.nix:62]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/networking/netkit/tftp/default.nix [hunk 44]
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     description = "Netkit TFTP client and server";
     homepage = "ftp://ftp.uk.linux.org/pub/linux/Networking/netkit/";
     license = lib.licenses.bsdOriginal;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/networking/netkit/tftp/default.nix:24]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/networking/olsrd/default.nix [hunk 45]
```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
     description = "Adhoc wireless mesh routing daemon";
     license = lib.licenses.bsd3;
     homepage = "http://olsr.org/";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/networking/olsrd/default.nix:33]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/networking/pdnsd/default.nix [hunk 46]
```diff
@@ -32,6 +32,6 @@ stdenv.mkDerivation rec {
     homepage = "http://members.home.nl/p.a.rombouts/pdnsd";
     license = licenses.gpl3Plus;
     platforms = platforms.unix;
-    maintainers = with maintainers; [viric];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/networking/pdnsd/default.nix:35]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/networking/pwnat/default.nix [hunk 47]
```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
     homepage    = "http://samy.pl/pwnat/";
     description = "ICMP NAT to NAT client-server communication";
     license     = lib.licenses.gpl3Plus;
-    maintainers = with maintainers; [viric];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/networking/pwnat/default.nix:37]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/networking/udptunnel/default.nix [hunk 48]
```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
     homepage = "https://code.google.com/archive/p/udptunnel/";
     description = "Tunnels TCP over UDP packets";
     license = lib.licenses.gpl3Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/networking/udptunnel/default.nix:22]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/security/ccrypt/default.nix [hunk 49]
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     homepage = "https://ccrypt.sourceforge.net/";
     description = "Utility for encrypting and decrypting files and streams with AES-256";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/security/ccrypt/default.nix:20]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/system/fakeroot/default.nix [hunk 50]
```diff
@@ -79,7 +79,7 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://salsa.debian.org/clint/fakeroot";
     description = "Give a fake root environment through LD_PRELOAD";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/system/fakeroot/default.nix:82]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/system/gt5/default.nix [hunk 51]
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     description = "Diff-capable 'du' browser";
     homepage = "https://gt5.sourceforge.net/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/system/gt5/default.nix:24]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/system/idle3tools/default.nix [hunk 52]
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     homepage = "https://idle3-tools.sourceforge.net/";
     description = "Tool to get/set the infamous idle3 timer in WD HDDs";
     license = lib.licenses.gpl3;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/system/idle3tools/default.nix:20]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/system/tm/default.nix [hunk 53]
```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
     description = "Terminal mixer - multiplexer for the i/o of terminal applications";
     homepage = "http://vicerveza.homeunix.net/~viric/soft/tm";
     license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/system/tm/default.nix:25]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/system/ts/default.nix [hunk 54]
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     homepage = "http://vicerveza.homeunix.net/~viric/soft/ts";
     description = "Task spooler - batch queue";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/system/ts/default.nix:24]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/text/mairix/default.nix [hunk 55]
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl2Plus;
     description = "Program for indexing and searching email messages stored in maildir, MH or mbox";
     mainProgram = "mairix";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/text/mairix/default.nix:24]:
```suggestion
    maintainers = [ ];
```

File: pkgs/tools/text/multitran/mtutils/default.nix [hunk 56]
```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
     description = "Multitran: simple command line utilities for dictionary maintenance";
     mainProgram = "mtquery";
     license = lib.licenses.gpl2Only;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
Comment (Mic92) [inline pkgs/tools/text/multitran/mtutils/default.nix:32]:
```suggestion
    maintainers = [ ];
```

File: pkgs/applications/audio/lingot/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/applications/audio/lingot/default.nix:49]: Mhm. I was hoping, I could just commit these straight to the branch, it it seems like github no longer allows me to add those.

**Comment by Mic92** [comment]
I will fix empty maintainer lists using the `with` expression in a follow up PR.

**Comment by AndersonTorres** [comment]
> As the first person to encounter the unused with error due to this, can you share your thoughts on this? 😄🎙️

I have mixed feelings about this.

In a first sight, I left the packages that way because a not so long time ago it was recommended - something about "it would be easier to add new maintainers in the future".

On the other hand, now that CI reports this as an error, there is no more room to never-ending discussions.

The net result of being shot by an error was positive!

**Comment by Mic92** [comment]
https://github.com/NixOS/nixpkgs/pull/330664

File: pkgs/applications/audio/lingot/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/applications/audio/lingot/default.nix]: This is a bad problem from GitHub: it does not allow edits from upstream maintainers when the PR comes from an organization.

**Comment by Mic92** [comment]
I think overall having detection for unused with statements is quite useful, while having an exception just for `maintainer` fields would make it only more complex.

**Comment by matthiasbeyer** [comment]
The discussion moved away from the actual topic of the PR AFAICS... how about we merge this?

**Comment by AndersonTorres** [comment]
> I will fix empty maintainer lists using the `with` expression in a follow up PR.

I can solve this with a bit of emacs-fu. Now CI is happy!
Nonetheless the cleanup will be needed in other expressions...

**Review by matthiasbeyer — APPROVED** [review]

**Comment by Mic92** [comment]
I thought, I pressed the merge button already... must have timed out.

**Comment by bendlas** [comment]
This removal PR has already been merged, despite not being grounded within community consensus. See https://github.com/NixOS/nixpkgs/pull/337478#issuecomment-2310779183

@viric  be sure that we value your involvement and any time you may have to spare, helping with the packages you've maintained and others as well, and if you decide to come back, please don't hesitate to be added to maintainers again.

**Pull request merged**